### PR TITLE
trivial: add device tests for Corsair devices

### PR DESCRIPTION
**Pull request by dushko-devx**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [ *] Feature
- [ ] Documentation

Jest device tests for several devices

### CODE REVIEW

File: data/device-tests/corsair-sabre-pro.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          "version": "1.15.25",
+          "protocol": "com.corsair.bp",
+          "guids": [
+            "9896f0c3-6682-5392-80f6-0ba462918645"
+          ]
+        }
+      ]
+    }
+  ]
+}
```
Comment (hughsie) [inline data/device-tests/corsair-sabre-pro.json:30]: github gets upset when there's no final newline; can you fix those please

Comment (dushko-devx) [reply data/device-tests/corsair-sabre-pro.json]: done

Comment (dushko-devx) [reply data/device-tests/corsair-sabre-pro.json]: done

**Review by hughsie — APPROVED** [review]

**Comment by hughsie** [comment]
@dushko-devx you can merge this yourself I think now, correct?

**Comment by dushko-devx** [comment]
@hughsie thanks

**Pull request merged**